src/why3prop.ml

```diff
@@ -0,0 +1,127 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Extra
+
+exception NoGoalTranslation
+
+(* a map from lambdapi terms to Why3 constants *)
+type cnst_table = (term * Why3.Term.lsymbol) list
+
+(* a map from labels to Why3 terms. *)
+type ctxt_labels = (string * Why3.Term.term) list
```

Comment:
Why not using StrMap.t?

Reply:
Fixed in e963c2e8677e9bfa6608c2268a47499eb15d49a4.